### PR TITLE
postgresql 14.0

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
only test build failure for `osm2pgsql`

**Comment by pthariensflame** [comment]
What needs to be done to help this?

**Comment by branchvincent** [comment]
we just need to fix the failing test.

```
==> Testing osm2pgsql (again)
==> /usr/local/Cellar/osm2pgsql/1.5.1/bin/osm2pgsql /dev/null 2>&1
Error: osm2pgsql: failed
An exception occurred within a child process:
  Minitest::Assertion: Expected /Connecting\ to\ database\ failed:\ could\ not\ connect\ to\ server/ to match "2021-10-01 13:00:05  osm2pgsql version 1.5.1\n2021-10-01 13:00:05  ERROR: Connecting to database failed: connection to server on socket \"/tmp/.s.PGSQL.5432\" failed: No such file or directory\n\tIs the server running locally and accepting connections on that socket?\n.\n".
```

it seems like the postgres error message just changed slightly, so i'll update the test

File: Formula/postgresql@13.rb [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+  livecheck do
+    url "https://ftp.postgresql.org/pub/source/"
+    regex(%r{href=["']?v?(13(?:\.\d+)+)/?["' >]}i)
+  end
+
+  keg_only :versioned_formula
+
+  # https://www.postgresql.org/support/versioning/
+  deprecate! date: "2025-11-13", because: :unsupported
```
Comment (SMillerDev) [inline Formula/postgresql@13.rb:16]: This should actually be disabled on that date. Maybe we deprecate it a year before?

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**